ci/llvm-gcov.sh

```diff
@@ -1,2 +1,21 @@
 #!/bin/bash
-exec "$LLVM4$LLVM5$LLVM6/bin/llvm-cov" gcov "$@"
+
+if [ -n "$LLVM7" ]; then
```

Comment:
Should we update documentation online saying that LLVM7 is also supported? I guess I checked it yesterday, and it was only 4/5/6.

Reply:
Well, we never tested with LLVM7, so it's too early to claim support. However, eventually when we add the support, this script won't have to be modified.